new_arrivals_chi/app/authorize_routes.py

```diff
@@ -16,7 +16,7 @@
     * post_change_password - Executes change password logic.
 
 Last updated:
-@Author: Madeleine Roberts @MadeleineKRoberts
+@Author: Federico Dominguez Molina @FedericoDM
```

Comment:
I'd just add a copyright for the group instead of updating this every time. GitHub will keep track of that history, and it makes it so you don't have to remember every field like the forgotten date below.